issues.py

```diff
@@ -5,24 +5,36 @@
 import json
 import hmac
 
-URL = os.getenv("GIT_REPO_URL")
-assert URL != None
+LGTM_URL = os.getenv("LGTM_WEBHOOK_URL")
+assert LGTM_URL != None
 
-ACCESS_TOKEN = os.getenv("GIT_ACCESS_TOKEN")
-assert ACCESS_TOKEN != None
+GITHUB_URL = os.getenv("GIT_REPO_URL")
+assert GITHUB_URL != None
 
-KEY = os.getenv("LGTM_SECRET", "").encode("utf-8")
+GITHUB_BOT_USERNAME = os.getenv("GIT_BOT_USERNAME")
+assert GITHUB_BOT_USERNAME != None
+
+GITHUB_TOKEN = os.getenv("GIT_ACCESS_TOKEN")
+assert GITHUB_TOKEN != None
+
+KEY = os.getenv("SECRET", "").encode("utf-8")
 assert KEY != "".encode("utf-8")
 
-headers = {
-    "content-type": "application/json",
-    "Authorization": "Bearer %s" % ACCESS_TOKEN,
-}
+sess_github = requests.Session()
+sess_github.headers.update(
+    {"content-type": "application/json", "Authorization": "Bearer %s" % GITHUB_TOKEN}
+)
+
+sess_lgtm = requests.Session()
+sess_lgtm.headers.update({"content-type": "application/json; charset=utf-8"})
```

Comment:
why the `; charset=utf-8` in the content-type? `application/json` is `utf-8` by default if I am not mistaken.